src/mono/msbuild/AppleAppBuilder/Templates/main-console.m

```diff
@@ -80,9 +80,10 @@ - (void)viewDidLoad {
     NSString* nsstr = [NSString stringWithUTF8String:strdup(value)];
     dispatch_async(dispatch_get_main_queue(), ^{
         logLabel.text = [logLabel.text stringByAppendingString:nsstr];
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [logLabel scrollRangeToVisible: NSMakeRange(logLabel.text.length -1, 1)];
-        });
+        CGRect caretRect = [logLabel caretRectForPosition:logLabel.endOfDocument];
+        [logLabel scrollRectToVisible:caretRect animated:NO];
+        [logLabel setScrollEnabled:NO];
+        [logLabel setScrollEnabled:YES];
     });
```

Comment:
Some hack to automatically scroll to the bottom in UITextView

Reply:
why do you need to toggle setScrollEnabled? is the scrollRectToVisible not enough?

Reply:
@akoeplinger I don't know but without it doesn't work properly, I tried different hacks from SO this one works as expected (https://stackoverflow.com/questions/19124037/scroll-to-bottom-of-uitextview-erratic-in-ios-7/20989956#20989956)